Makefile-az.mk

```diff
@@ -26,7 +26,7 @@ az-all-custom-vnet: az-login az-create-workload-msi az-mkaks-custom-vnet az-crea
 az-all-savm:    az-login az-mkaks-savm az-perm-savm az-patch-skaffold-azure az-build az-run az-run-sample ## Provision the infra (ACR,AKS); build and deploy Karpenter; deploy sample Provisioner and workload - StandaloneVirtualMachines
 
 az-login: ## Login into Azure
-	az login
+	az account show -o none || az login
```

Comment:
Is this an intended change for this PR?

Reply:
Yes. While not strictly speaking part of the PR functionality, it addresses a long-standing problem with our makefile workflows: the fact that login is performed even if one is already logged in.